client/packages/programs/src/JsonForms/common/components/Spacer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { ControlProps, rankWith, uiTypeIs } from '@jsonforms/core';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import { Box, Divider } from '@openmsupply-client/common';
+import { z } from 'zod';
+import { useZodOptionsValidation } from '../hooks/useZodOptionsValidation';
+
+const Options = z
+  .object({
+    margin: z.number().optional(),
+    hideDivider: z.boolean().optional(),
+  })
+  .strict()
+  .optional();
+
+type Options = z.infer<typeof Options>;
+
+export const spacerTester = rankWith(3, uiTypeIs('Spacer'));
+
+const UIComponent = ({ uischema, errors }: ControlProps) => {
+  // Validates the option
+  const { errors: zErrors, options: schemaOptions } = useZodOptionsValidation(
+    Options,
+    uischema.options
+  );
+
+  const error = !!errors || !!zErrors;
+  const margin = schemaOptions?.margin ?? 20;
+  const showDivider = !schemaOptions?.hideDivider;
+
+  if (error) return <Divider margin={margin} color="red" />;
```

Comment:
Seems a bit overkill to have an error state for a passive element like this, but can't hurt, right?

Reply:
Yes cool, easy to add the error text in this case? doesn't has to look nice because it should be for devs only...

Reply:
> Yes cool, easy to add the error text in this case? doesn't has to look nice because it should be for devs only...

fb9e4a107